src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/EventToCommandBehaviorGeneric.shared.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	/// <summary>
+	/// This <see cref="EventToCommandBehavior"/> cast the sender object to a specific type defined by the user.
+	/// </summary>
+	/// <typeparam name="TType">The type that you want to receive in your <see cref="Xamarin.Forms.Command"/> </typeparam>
+	public sealed class EventToCommandBehavior<TType> : EventToCommandBehavior
+	{
+		protected override void OnTriggerHandled(object? sender = null, object? eventArgs = null)
+		{
+			var parameter = CommandParameter
+				?? EventArgsConverter?.Convert(eventArgs, typeof(object), null, null)
+				?? eventArgs;
+
+			if (parameter is not TType)
+			{
+				// changing it to the default value to avoid a cast exception
+				parameter = default(TType);
+			}
```

Comment:
```suggestion
			// changing it to the default value to avoid a cast exception
			if (parameter is not TType)
				parameter = default(TType);
```

Reply:
We're not using braces? I recall some recent PRs that we added the braces for one line

Reply:
I don't know really. Need to raise this question during stand-up )